addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/ZoneHandler.java

```diff
@@ -23,6 +23,7 @@
 
 public class ZoneHandler extends AbstractOmnilinkHandler {
     private Logger logger = LoggerFactory.getLogger(ZoneHandler.class);
+    private ZoneStatus zoneStatus;
```

Comment:
volatile may not be the way to go, but I think we will need some form of memory barrier to gaurentee correctness.   Volatile might be too weak.

I am on mobile right now, but I will review the code tomorrow and provide an explanation of why I think a memory barrier is or is not warranted in this case.

Hopefully we don't need one.